components/admin_console/admin_navbar_dropdown/index.js

```diff
@@ -16,6 +16,7 @@ function mapStateToProps(state) {
     return {
         locale: getCurrentLocale(state),
         teams: getMyTeams(state),
+        siteName: state.config.SiteName,
```

Comment:
@reflog This is crashing the app on master when one clicks the hamburger menu from the system console.

```suggestion
        siteName: state.entities.general.config.SiteName,
```

Reply:
I know :( Already submitted a PR https://github.com/mattermost/mattermost-webapp/pull/2877